guides/common/modules/proc_renaming-smart-proxy.adoc

```diff
@@ -62,9 +62,9 @@ For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-remove} katello-ca-consumer*
+# {client-package-remove-el8} katello-ca-consumer*
 
-# {project-package-install} http://_new-{smartproxy-example-com}_/pub/katello-ca-consumer-latest.noarch.rpm
+# {client-package-install-el8} http://_new-{smartproxy-example-com}_/pub/katello-ca-consumer-latest.noarch.rpm
```

Comment:
The replacement is correct, however, shouldn't this step suggest using Global registration instead of a deprecated registration method? For another PR.

Reply:
Yeah, I had the same thought! Noted for a follow-up PR!